proprietary/intel-mkl/default.nix

```diff
@@ -10,16 +11,20 @@ let
     name = "intel-mkl";
     unpackPhase = ''echo "no sources"'';
     dontBuild = true;
+    dontStrip = true;
     installPhase = ''
       mkdir -p $out/lib;
 
       pushd ${compiler_path}/mkl/lib/intel64
-      for i in ./* ; do
-        ln -s "${compiler_path}/mkl/lib/intel64/$i" "$out/lib/$i";
+      for i in * ; do
+        echo "Nixify $i under ${compiler_path}/mkl/lib/intel64"
+        cp -f -L  "${compiler_path}/mkl/lib/intel64/$i" "$out/lib/$i" || true
+        ${patchelf}/bin/patchelf --set-interpreter ${stdenv.glibc}/lib/ld-linux-x86-64.so.2 $out/lib/$i || /bin/true
```

Comment:
I will check tomorrow how this thing is handled in spack (I haven't seen similar things)